src/app/public/modules/file-attachment/file-attachment.component.spec.ts

```diff
@@ -1083,6 +1083,14 @@ describe('File attachment', () => {
       expect(fixture.nativeElement).toBeAccessible();
     });
   }));
+
+  it('should pass accessibility when label changes', async(() => {
+    fixture.componentInstance.labelText = 'Something different';
```

Comment:
We say `when label changes` but are only testing the "after". I know we are testing the "before" in the test before but would you mind calling that out in a quick code comment there?

Reply:
Updated the comments to be more clear.